### ISSUE TITLE
Slider buttons are almost invisible

### ISSUE THREAD

**Issue by robines** [Samfundet/Samfundet4#1023]
They should be a bit less transparent, I can barely see them

<img width="1495" alt="image" src="https://github.com/Samfundet/Samfundet4/assets/16273164/c74ed2e4-7857-48cf-8e81-54e28692f6b4">

**Comment by robines** [comment]
@snorrekr Agreed, but would be nice to make the buttons bigger then, taking the whole vertical space of the slider. And maybe add some sort of hover effect to make it obvious that something will happen if you click

**Comment by snorrekr** [comment]
Will do.

**Comment by aTrueYety** [comment]
Buttons dont work on small phone
